tests/schema/tests.py

```diff
@@ -1651,6 +1651,23 @@ def test_order_index(self):
                 index_name = index_name.upper()
             self.assertIndexOrder(Author._meta.db_table, index_name, ['ASC', 'DESC'])
 
+    @skipIfDBFeature('supports_index_on_text_field')
+    def test_mysql_non_supported_db_index_fields(self):
+        """
+        MySQL do not support indexing of blob/text/json fields
+        """
+        class NoteInfoIndexed(Model):
+            info = TextField(db_index=True)
+
+            class Meta:
+                app_label = 'schema'
+                apps = new_apps
```

Comment:
You can move `NoteInfoIndexed` to `schema/models.py`. Table will be created only in this test (see [[1]](https://github.com/django/django/pull/8118/files#diff-888b45a0a8f38ee67e8d22403cf994dbR1667)).